outputs.tf

```diff
@@ -37,3 +37,13 @@ output "kubeconfig" {
   description = "location of kubeconfig"
   value       = local.kubeconfig_path
 }
+
+output "kms_key_id" {
```

Comment:
do we always say id when its an arn?
and I see we use key in the var. just need to be consistent so we know what the heck it is.

Reply:
Oddly, yes, and even in external documentation:
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/b926e1ee4551e60b44c9b5a887409a8feaf91d86/docs/parameters.md?plain=1#L16
https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingKMSEncryption.html#:~:text=key%20using%20the-,x%2Damz%2Dserver%2Dside%2Dencryption%2Daws%2Dkms%2Dkey%2Did,-header.%20If%20the

I was trying to disambiguate with the input since it can be an ID, ARN, alias (whatever the data source can lookup).

Reply:
well that link shows key-id as the last part of an ARN. 

notice key_id vs key_arn
https://github.com/terraform-aws-modules/terraform-aws-kms/blob/master/outputs.tf#L5

Reply:
Ok yeah I totally misread it before testing it, I'll add two outputs.

Reply:
Nevermind, yeah it's definitely the ARN. The wording is awful, but I think it's the ARN everywhere.